packages/charts-components/templates/webpack/README.md

```diff
@@ -0,0 +1,135 @@
+          to: "./arcgis-charts/t9n"
+        }
+      ],
+    }),
+  ],
+};
+```
+
+#### JS
+
+We imported the components we need for the coding components by following [Stencil's instructions for integrating components without a JavaScript framework](https://stenciljs.com/docs/javascript).
+
+```
+import { defineCustomElements as defineChartsElements } from "@arcgis/charts-components/dist/loader";
+
+defineChartsElements(window, {
+  resourcesUrl: "../arcgis-charts/"
+});
+```
+
+We use [`src/index.js`](./src/index.js) to load our data, define our custom elements, and utilize various kinds of properties in the editor. It is a must to define both the Charts elements.
+
+#### CSS
+
+You can find all the necessary styling in [`src/index.css`](./src/index.css). 
+
+#### HTML
+
+The generation of our `index.html` was simplified by using the HtmlWebpackPlugin in the webpack configuration file.
+
+```
+// webpack.config.js
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+  plugins: [
+     new HtmlWebPackPlugin({
+      title: "Charts components",
+      chunksSortMode: "none",
+      meta: {
+        viewport:
+          "width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0, user-scalable=no",
+      },
+      templateContent: `
+        <!DOCTYPE html>
+          <html dir="ltr" lang="en">
+            <head>
+              <meta charset="utf-8">
+            </head>
+            <body>
+              <div class="flex-container">
+                <div class="chart-container">
+                  <arcgis-charts-bar-chart class="half-screen-chart"></arcgis-charts-bar-chart>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-bar-line-chart class="half-screen-chart"></arcgis-charts-bar-line-chart>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-box-plot class="half-screen-chart"></arcgis-charts-box-plot>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-histogram class="half-screen-chart"></arcgis-charts-histogram>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-line-chart class="half-screen-chart"></arcgis-charts-line-chart>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-pie-chart class="half-screen-chart"></arcgis-charts-pie-chart>
+                </div>
+                <div class="chart-container">
+                  <arcgis-charts-scatter-plot class="half-screen-chart"></arcgis-charts-scatter-plot>
+                </div>
+              </div>
+            </body>
+          </html>`,
+    }),
+  ],
+};
+```
+
+## Resources
+
+[Webpack's getting started guide](https://webpack.js.org/guides/getting-started/)
+
+[Webpack 5's `experiments` option](https://webpack.js.org/configuration/experiments/)
```

Comment:
Double checking, are experiments being used in the webpack.config?

Reply:
It is not, I can remove this from the README, I see the same section in the coding components README for webpack. 

https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/blob/a755d8c29d224348ba55a98ff04b77e4f3ff2c27/packages/coding-components/templates/webpack/readme.md?plain=1#L91

Reply:
> It is not,

Gotcha, yep let's go ahead and remove it.